classes/locations.php

```diff
@@ -778,23 +788,23 @@ function ajax_save_lat_lng() {
 
 			// If we're missing a var, return false.
 			if ( empty( $_POST['sm_lat'] ) || empty( $_POST['sm_lng'] ) || empty( $_POST['sm_id'] ) ) {
-				die( __( "It doesn't look like that worked either. Please try again later.", 'SimpleMap' ) );
+				die( __( "It doesn't look like that worked either. Please try again later.", 'simplemap' ) );
 			}
 
 			// Save original lat for roleback if lng update fails.
 			$orig_lat = get_post_meta( absint( $_POST['sm_id'] ), 'location_lat', esc_attr( $_POST['sm_lat'], true ) );
 
 			// Update or return false.
 			if ( ! update_post_meta( absint( $_POST['sm_id'] ), 'location_lat', esc_attr( $_POST['sm_lat'] ) ) ) {
-				die( __( "It doesn't look like that worked either. Please try again later.", 'SimpleMap' ) );
+				die( __( "It doesn't look like that worked either. Please try again later.", 'simplemap' ) );
 			}
 			if ( ! update_post_meta( absint( $_POST['sm_id'] ), 'location_lng', esc_attr( $_POST['sm_lng'] ) ) ) {
 				update_post_meta( absint( $_POST['sm_id'] ), 'location_lat', esc_attr( $orig_lat ) );
-				die( __( "It doesn't look like that worked either. Please try again later.", 'SimpleMap' ) );
+				die( __( "It doesn't look like that worked either. Please try again later.", 'simplemap' ) );
 			}
 
 			// If we made it here, we're golden.
-			die( __( 'It looks like that worked!', 'SimpleMap' ) );
+			die( __( 'It looks like that worked!', 'simplemap' ) );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->